setup/mail-postfix.sh

```diff
@@ -215,6 +215,83 @@ tools/editconf.py /etc/default/postgrey \
 tools/editconf.py /etc/postfix/main.cf \
 	message_size_limit=134217728
 
+# ### MTA-STS - SMTP Mail Transfer Agent Strict Transport Security - SETUP
+# See: https://github.com/mail-in-a-box/mailinabox/pull/1556
+# create the MTA-STS policy
+hide_output mkdir -p /var/lib/mailinabox/
```

Comment:
There's no need for hide_output here. `mkdir -p` never has output.

Reply:
Let's also clarify in the comment that this part is what announces the local MTA-STS policy to senders and the file is served by nginx on mta-sts subdomains (actually all domains, but that's fine).

Reply:
> There's no need for hide_output here. `mkdir -p` never has output.

Similarly, is the hide_output in the "hide_output /bin/systemctl start postfix-mta-sts.service" line necessary?  It probably is in the "enable" line as this often creates chatter when it is created, I'm not sure what happens if the service would fail to start (starting is silent).

Let me know if you would like any corrections to the additional comment.